docs/specification/appendix.md

```diff
@@ -65,15 +65,43 @@ Schema `encoding` may only be omitted for self-describing message encodings such
 
 ### ros1msg
 
-- `name`: A valid [package resource name](http://wiki.ros.org/Names#Package_Resource_Names), e.g. `sensor_msgs/PointCloud2`
+- `name`: A valid [package resource name](http://wiki.ros.org/Names#Package_Resource_Names), e.g. `sensor_msgs/PointCloud2`. The package resource name does not include the file extension.
```

Comment:
@defunctzombie the package resource name definition is linked here.

Reply:
This is for `ros1msg`. It might be worth being explicit for `ros2idl` as well. I would almost operate like ros2 is a separate system from ros1.